site/docs/reference/Connectors/capture-connectors/google-analytics.md

```diff
@@ -1,10 +1,12 @@
-# Google Analytics
+# Google Universal Analytics
 
-This connector captures data from a view in Google Analytics.
+This connector captures data from a view in Google Universal Analytics.
 
-:::caution
-This connector supports Universal Analytics.
-Support for Google Analytics 4 is coming soon.
+:::info
+This connector supports Universal Analytics, not Google Analytics 4.
+(We apologize for the misleading name of the Docker image below.)
```

Comment:
I'll make an attempt to rename the docker image

Reply:
Okay the connector for this one is now called `google-analytics-ua`

Reply:
Hmm... I don't see it renamed (https://github.com/orgs/estuary/packages) and that URL doesn't work for me

Reply:
Never mind, was just missing "source" from the URL :) Would be good to have it listed on the packages page under the alias but can confirm I'm able to access the connector's page.